jvm/src/main/scala/io/glutenproject/GlutenPlugin.scala

```diff
@@ -144,6 +144,11 @@ private[glutenproject] object GlutenPlugin {
       nativeConfMap.put(
         GlutenConfig.HIVE_EXEC_ORC_COMPRESS, conf.get(GlutenConfig.SPARK_HIVE_EXEC_ORC_COMPRESS))
     }
+    if (conf.contains(GlutenConfig.GLUTEN_CLICKHOUSE_CONFIF_FILE)) {
+      nativeConfMap.put(
+        GlutenConfig.GLUTEN_CLICKHOUSE_CONFIF_FILE,
```

Comment:
I think the config file priority should be lower than spark conf, the spark conf in runtime scope should have the highest priority. Do you think so?

Reply:
I agree with you. It's similiar in clickhouse,  session settings > user settings > config settings.

Reply:
Now we have update the implementation to setup configurations for clickhouse backend.
please to see [ kyligence/clickhouse#118](https://github.com/Kyligence/ClickHouse/pull/118)